pkg/oc/admin/image/verify-signature.go

```diff
@@ -143,8 +159,6 @@ func (o *VerifyImageSignatureOptions) Complete(f *clientcmd.Factory, cmd *cobra.
 
 	// We need the current user name so we can record it into an verification condition and
 	// we need a bearer token so we can fetch the manifest from the registry.
-	// TODO: Add support for external registries (currently only integrated registry will
```

Comment:
In this context, the *external* means the hosted outside of cluster, right? As such, this looks still valid.